package.json

```diff
@@ -183,19 +186,26 @@
 	},
 	"dependencies": {
 		"@kubernetes/client-node": "^0.17.1",
+		"@patternfly/react-core": "^4.276.8",
+		"@patternfly/react-styles": "^4.92.6",
+		"@patternfly/react-table": "^4.113.0",
+		"@types/d3-shape": "^3.1.1",
```

Comment:
I noticed you added the patternfly dependencies to both the root project.json and the webui one. Is this necessary?

Reply:
to be honest, i am not sure. 
only after adding it to both spaces it worked. 
but it could that should have added only part of it

Reply:
I think they should go only in webui, I also see other dependencies that exist only in webui and not the root, and it works well. Can you please try building while keeping this only in webui?